File: rpg/schemas/worldline.py
"""schemas.worldline — 世界线变量管理路由请求模型。"""
from __future__ import annotations

from typing import Optional

from schemas._common import _BaseRequest


class WorldlineVariableRequest(_BaseRequest):
    key: str | None = ""
    value: str | None = ""


class WorldlineVariableRemoveRequest(_BaseRequest):
    key: str | None = ""

File: rust/crates/rpg-schemas/src/worldline.rs
//! schemas.worldline — 世界线变量管理路由请求模型。
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize, Default)]
pub struct WorldlineVariableRequest {
    #[serde(default)]
    pub key: Option<String>,
    #[serde(default)]
    pub value: Option<String>,
}

#[derive(Debug, Clone, Serialize, Deserialize, Default)]
pub struct WorldlineVariableRemoveRequest {
    #[serde(default)]
    pub key: Option<String>,
}
